website/pages/api-docs/acl/auth-methods.mdx

```diff
@@ -61,6 +61,10 @@ The table below shows this endpoint's support for
 
   This must be set to a nonzero value for `type=oidc`.
 
+- `TokenType` `(string: "")` - Defines the kind of token that this auth method
+  should produce. This can be either `"local"` or `"global"`. If empty the
+  value of `"local"` is assumed. Added in Consul 1.8.0.
```

Comment:
I think it would be helpful to clarify what local and global mean. Maybe something like (assuming scope is an appropriate term) :

> `TokenScope` - defines the scope of the token produced by this auth method. `local` scope token are only valid in the datacenter in which they are created. `global` scope tokens are valid in all connected datacenters.

Reply:
One argument for leaving it as TokenType is that if in the future we allow for stateless tokens (JWT based possibly) then its just another type to support here. Otherwise we would need a token type (traditional vs stateless) and the scope if the type is traditional

Reply:
Ugh I just realized `structs.ACLToken` has a field named `Type` for the v1 acl tokens:
```
const (
	// ACLTokenTypeClient tokens have rules applied
	ACLTokenTypeClient = "client"

	// ACLTokenTypeManagement tokens have an always allow policy, so they can
	// make other tokens and can access all resources.
	ACLTokenTypeManagement = "management"

	// ACLTokenTypeNone
	ACLTokenTypeNone = ""
)
```
Calling _this_ a "token type" might end up confusing someone later?

Reply:
@mkeeler

Reply:
done